favorite_files.py

```diff
@@ -122,6 +140,11 @@ def add(self, names, group_name=None):
         if added:
             # Save if files were added
             Favs.save(True)
+            if len(names) == 1 and settings().get('always_ask_alias', False):
+                # pass index as -1 because json will be sorted and we can't know it
+                # it will be searched in the called function
+                Favs.prompt_for_alias(-1, names[0], group_name)
```

Comment:
UI stuff should happen at this layer, and but acquiring aliases can be occur through `Favs`.